File: app/src/main/java/com/main/climbingdiary/database/entities/Sector.java
package com.main.climbingdiary.database.entities;

import org.chalup.microorm.annotations.Column;

import lombok.Data;

@Data
public class Sector {
    @Column("gebiet") private Integer area_name;
    @Column("name") private String name;
    @Column("id") private int id;
    @Column("lat") private double lat;
    @Column("lng") private double lng;
}

File: app/src/main/java/com/main/climbingdiary/database/entities/Sector.kt
package com.main.climbingdiary.database.entities

import org.chalup.microorm.annotations.Column

data class Sector(
    @Column("gebiet") var area:Int,
    @Column("name") var name: String,
    @Column("id") var id: Int =0,
    @Column("lat") var lat: Double = 0.0,
    @Column("lng") var lng: Double = 0.0
)
